content/blog/2023-08-15-ITs/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Interactive tool collection is growing"
+date: "2023-08-15"
+tease: "New/improved interactive tools are available"
+hide_tease: false
+authors: 'Tunc Kayikcioglu'
+authors_structured:
+- github: tuncK
+tags: [EU]
+subsites: [all-eu]
+main_subsite: eu
+---
+
+
+August more or less marks the end of the summer semester here in southwestern Germany. Exquisite time to enjoy the nature before it is too late, but also a good time to finish up leftover tasks and contemplate on our progress and make plans for the future. At the top of this list is this way overdue blog post regarding our work on Galaxy interactive tools, which allow users to visually communicate with software during its execution on Galaxy.
+
+
+## What are interactive tools (IT)?
+
+For a non-interactive tool execution, the user defines all inputs before the job execution and has access to the results after the job finishes. During the execution phase of the job, the user does not and cannot interact with the system. For majority of the scientific software, this model is actually sufficient, because they are commandline-only, the computation follows a pre-written recipe in the script and a user input during the computation is not needed at all.
```

Comment:
I am not sure this constitutes the only role of ITs. This is just an extra option/luxury that the users get.

For me IT's or VREs are more like the below:

ITs or VREs (Virtual research environments) provide researchers with digital spaces to conduct experiments, analyze data, and collaborate seamlessly. VREs such as Jupyter Notebook and Rstudio empower users with interactive coding environments, enabling real-time manipulation of code, visualizations, and data analysis and fostering iterative exploration. Thus, enhancing efficiency and reproducibility within these dynamic research landscapes.